src/table/utils/announcement-factory.js

```diff
@@ -35,7 +46,21 @@ export default function announcementFactory(rootElement, translator, junkCharIdx
     if (hasJunkChar % 2) notation += ` ­`;
     hasJunkChar++;
 
-    const announceElement = rootElement.querySelector('#sn-table-announcer');
+    const announceElement01 = rootElement.querySelector('#sn-table-announcer--01');
+    const announceElement02 = rootElement.querySelector('#sn-table-announcer--02');
+
+    let announceElement = null;
+    if (lastAnnounceElement === null) {
+      announceElement = announceElement01;
+      lastAnnounceElement = announcerElements.first;
+    } else if (lastAnnounceElement && lastAnnounceElement === announcerElements.first) {
+      announceElement = announceElement02;
+      lastAnnounceElement = announcerElements.second;
+    } else {
+      announceElement = announceElement01;
+      lastAnnounceElement = announcerElements.first;
+    }
```

Comment:
Had trouble understanding this. The following should do the same thing
```suggestion
    if (lastAnnounceElement === announcerElements.first) {
      announceElement = announceElement02;
      lastAnnounceElement = announcerElements.second;
    } else {
      announceElement = announceElement01;
      lastAnnounceElement = announcerElements.first;
    }
```

Reply:
✅.  Fixed